packages/portfolio/src/data-access/use-create-and-send-spl-transaction.tsx

```diff
@@ -25,7 +25,13 @@ export function useCreateAndSendSplTransaction({ network }: { network: Network }
       mint: string
       sender: KeyPairSigner
     }) => {
-      return createAndSendSplTransaction(client, { amount, decimals, destination, mint, sender })
+      return createAndSendSplTransaction(client, {
```

Comment:
Same here, invocation of `address()` moved to FE, will move further up to the form.